dist/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>To-Do List</title>
+<script defer src="main.js"></script></head>
+<body>
+   <section id="to-do">
+

```

Comment:
I suggest that you use blank lines only when separating large or logical code blocks.  This would make the code look clean and professional. Please read more from [this documentation](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) on HTML best practices